manifest.json

```diff
@@ -186,6 +186,22 @@
 				"libs/jquery-3.4.1.min.js",
 				"dist/roll20.js"
 			]
+		},
+		{
+			"matches": [
+				"*://1199271093882589195.discordsays.com/*"
```

Comment:
Assuming discord activity IDs static/reserved (or at least reasonably so):

Are there other proxies that the activity might be served through? eg geolocal/app-specific/mweb

Reply:
Yes the Discord Activity IDs are static and I'm not aware of any other URLs where the activity will be served from.